lib/Client.php

```diff
@@ -271,6 +284,46 @@ private function send($key, $value, $type, $sampleRate)
     }
 
     /**
+     * @param $tags
+     * @return string
+     */
+    private function generateTagsMessage($tags)
+    {
+      // Nothing to return if no tags
+      if (empty($tags)) {
+        return '';
+      }
+
+      // Normalize string inputs into an array
+      if (!is_array($tags)) {
+        $tags = [$tags];
```

Comment:
looking at the test, it seems that it ends up being named the same thing as the value... can't honestly say i understand why, but you can disregard my comment.